cartridges/LINK_adyen/cartridge/templates/default/checkout/billing/creditCardForm.isml

```diff
@@ -1,121 +1,5 @@
+<input type="hidden" class="cardNumber" value="" id="cardNumber" >
```

Comment:
is the class necessary for the hidden input? Also if no value is assigned the value attribute can also be deleted